frontend/src/component/onboarding/sdkSnippets.ts

```diff
@@ -29,12 +29,13 @@ dotnet add package Newtonsoft.Json`,
 };
 
 export const initializeCodeSnippets: Record<SdkName, string> = {
-    Node: `import { initialize } from 'unleash-client';
+    Node: `const { initialize } = require('unleash-client');
 
 const unleash = initialize({
   url: '<YOUR_API_URL>',
   appName: 'unleash-onboarding-node',
   customHeaders: { Authorization: '<YOUR_API_TOKEN>' },
+  metricsInterval: 5000,
```

Comment:
I was waiting for a while to get it connected, thought it is disabled by default, but it seems default is 60 seconds and it was not responsive enough, that I expected. Making it 5 seconds, so metrics are sent sooner.

Reply:
We can even make it 1 second to see the results quicker